dcos/metronome.py

```diff
@@ -338,12 +338,31 @@ def kill_run(self, job_id, run_id):
         :type run_id: str
         :rtype: None
         """
-
+        if job_id is None or run_id is None:
+            raise DCOSException("job_id and run_id are required to kill a jobrun.")
         job_id = util.normalize_marathon_id_path(job_id)
         run_id = util.normalize_marathon_id_path(run_id)
         path = '/v1/jobs{}/runs{}/actions/stop'.format(job_id, run_id)
         self._rpc.http_req(http.post, path)
 
+    def get_queued_jobruns(self, job_id):
+        """Returns the content of the launch queue,
+        including the jobruns for each job_id which should be scheduled.
+
+        :returns: a list of jobruns to launch
+        :rtype: list of dict
+        """
+        response = self._rpc.http_req(http.get, 'v1/queue')
+        if job_id is not None:
+            deployments = [
+                deployment for deployment in response.json()
+                if job_id in deployment['jobId']
```

Comment:
I believe that in on strings is actually doing substring so this would return true for "jobId" in "jobId2" so I would actually prefer to compare on equals here